doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/knowledge-store-rest-api-projects-ref.adoc

```diff
@@ -0,0 +1,439 @@
+// To reuse this module, ifeval the title to be more specific as needed.
+
+[id='knowledge-store-rest-api-projects-ref_{context}']
+= Projects
+
+The Knowledge Store REST API supports the following endpoints for managing, building, and deploying projects in {CENTRAL}. The Knowledge Store REST API base URL is `\http://SERVER:PORT/{URL_COMPONENT_CENTRAL}/rest/`. All requests require basic HTTP Authentication or token-based authentication for the `rest-all` user role.
+
+[GET] /projects::
++
+--
+Returns all projects in {CENTRAL}.
```

Comment:
I don't think we support this, projects are related to spaces, you need to list all spaces to get this info.

Reply:
K. Will remove.